### PR TITLE
#164364266 Define Reports table model

### PR DESCRIPTION
**Pull request by Ndifreke**
**What does this PR do ?**
* Creates Reports model

**Description of Task to be completed ?**
* Create Model dependencies in test helpers 

**What are the relevant pivotal tracker story ?**
[#164364266](https://www.pivotaltracker.com/n/projects/2240406/stories/164364266)

### CODE REVIEW

File: server/models/reports.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      allowNull: false
+    },
+    message: {
+      type: DataTypes.TEXT,
+      allowNul: false
+    },
+    isResolved: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+      allowNul: false
```
Comment (obasajujoshua31) [inline server/models/reports.js]: please check the spelling here for allowNull

File: server/tests/helpers/dependencies.js [hunk 2]
```diff
@@ -44,3 +43,61 @@ export const updateProfileDependencies = async (email, password, userScope) => {
     token
   });
 };
+
+const destroyDependencies = function(depencies) {
```
Comment (obasajujoshua31) [inline server/tests/helpers/dependencies.js]: spelling dependencies

File: server/tests/helpers/dependencies.js [hunk 3]
```diff
@@ -44,3 +43,61 @@ export const updateProfileDependencies = async (email, password, userScope) => {
...
+  this.forEach(async depency => {
+    depency.destroy();
```
Comment (obasajujoshua31) [inline server/tests/helpers/dependencies.js]: dependency not depency

**Review by obasajujoshua31 — COMMENTED** [review]
please there are no test to test the associations

**Review by eokwukwe — CHANGES_REQUESTED** [review]
Please edit your commit header
`chore(setup): define Reports table model`
#### should be 
`chore(reportModel): define reports table model`.

It is important that the `<scope>` in the commit header represent exactly what the task is about. Your task is not a `setup` task. The `database` connection has already been setup and running. Your task is to create a `model` for the report table in the database.

Also, your commit does contain a commit body message. At least a message describing the task may be appropriate.

Please try to review the `wiki` for proper commit message convention.

**Comment by benfluleck** [comment]
Both of you are wrong the correct commit is 

`chore(setup-report-Model): Define the reports table model.` cc @eokwukwe @Ndifreke  🙇

**Review by eokwukwe — COMMENTED** [review]
The following statement was taken from our convention as stated in the `wiki`:

```
<type>(<scope>): <subject>
<BLANK LINE>
<body>
<BLANK LINE>
<footer>
```

> #####`<scope>` Scope can be anything specifying place of the commit change. For example events, kafka, userModel, authorization, authentication, loginPage, etc...
> 
> #####`<subject>` This is a very short description of the change.
> 
> use imperative, present tense: “change” not “changed” nor “changes” don't capitalize first letter no dot (.) at the end


```
feat(kafka): implement exactly once delivery

- ensure every event published to kafka is delivered exactly once
- implement error handling for failed delivery

[Delivers #130635935]
```


My comment was based on the above statement from the `wiki`.

**Review by eokwukwe — APPROVED** [review]

**Review by obasajujoshua31 — APPROVED** [review]

**Pull request merged**